Add conferences playbook (#88)
artsy/README: playbooks/conferences.md

---
title: Conferences
description: Attending or speaking at a conference
---

# Conferences

We encourage engineers to attend and speak at conferences. It is a great way to stay up-to-date with the industry,

> be inspired by challenges others are facing, develop personal career, and connect with the community. Here are some

Reviewer: ```suggestion
be inspired by challenges others are facing, develop one’s personal career, and connect with the community. Here are some
```

guidelines:

> ## Attending Conferecens

Reviewer: ```suggestion
## Attending Conferences
```

- We aim for each engineer to be able to attend a conference every year, subject to budget. The conference should

>   be related to individual's role or Artsy's challenges/technologies.

Reviewer: ```suggestion
  be related to the individual's role or Artsy's challenges/technologies.
```

Reviewer: In the past we’ve agreed to send engineers to courses/workshops that are great for the individual’s growth, but less directly for their current role. An example would be an intensive 5 day design course. Do we consider that beneficial to their _future_ role, an ‘Artsy challenge’, or something separate?

> - Please be mindful when expensing your travel. A good rule of thumb is to spend the money like your own. Refer to

Reviewer: ```suggestion
- Please be mindful when expensing your travel. A good rule of thumb is to spend Artsy’s money like you would your own. Refer to
```

  [Atlas][travel-and-accommodations-on-atlas] for more details and rate card.
- It's strongly recommended writing up take-aways and sharing with the team upon return, e.g. via [Lunch and

>   Learn][lunch-and-learn], [Show and Tell][show-and-tell], or a blog post.

Reviewer: I’d like to see another option that may be a little less daunting for people than speaking/blogging. E.g. Slacking some notes (in a public channel) with links to projects/docs that were discussed during a talk and @ some people that you know are dealing with that space is a good ROI as well imo.

Author: Yeah good point! This should not be intimidating or discouraging.

Reviewer 2: I agree that it can be lightweight! A list of take-aways and links to the best of the online content (decks/videos) could be enough, if shared with the most relevasnt distribution list of slack channel.

> ## Speaking at Confereces

Reviewer: ```suggestion
## Speaking at Conferences
```

- We encourage engineers to speak at conferences, and budget-wise the expense gets higher priority.
- If you need help preparing a conference talk, stop by [Ash][ash]'s weekly Writing Office Hours.

> - It's also recommended sharing the talk and lessons with the team.

Reviewer: I guess I should L&L on GraphQL errors! 😄

Author: I'd love to see it! 🍻

## Conference List

A growing list of related conferences:

- JSConf: https://jsconf.com/
- RailsConf: https://railsconf.com/
- React conferences: https://reactjs.org/community/conferences.html
- RubyConf: http://rubyconf.org/
- SmashingConf: https://smashingconf.com/
- Strange Loop: https://www.thestrangeloop.com/index.html

> - dotJS: https://www.dotjs.io/

Reviewer: These are all big and expensive confs, there’s a plethora of cheaper confs that are great too. Maybe it would be more helpful to link to curated lists of confs in various areas?

Author: Yeah these are starting points and we should definitely add good ones. I was thinking it makes sense to use this playbook as the canonical list when people are looking for engineering conferences to participate, and I'm also happy to link to other curated lists in various areas.

I was trying to group them by month/quarter throughout the year so people can plan, but it turned out not very easy. 😛
